src/components/__tests__/Alert.spec.ts

```diff
@@ -0,0 +1,40 @@
+import { createElement } from "react";
+import { shallow } from "enzyme";
+import { Alert } from "../Alert";
+
+describe("Alert", () => {
+
+    it("renders structure correctly", () => {
+        const message = "This is an error";
+        const alert = shallow(createElement(Alert, { bootstrapStyle: "danger", message }));
+
+        expect(alert).toBeElement(
+            createElement("div", { className: "alert alert-danger" }, message)
+        );
+    });
+
+    it("renders no structure when the alert message is not specified", () => {
+        const alert = shallow(createElement(Alert, { bootstrapStyle: "danger" }));
+
+        expect(alert).toBeElement(null);
+    });
+
+    it("shows green when bootstrap style is success", () => {
```

Comment:
sure...i have changed the test name but i think the others are more less irrelevant.